install.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+echo "This will install ATHEME, IRCD-SEVEN (charybdis) AND GMS!"
+echo
+echo "Read now the README"
+
+echo
+
+curl "https://raw.githubusercontent.com/freenode/gms/master/README" | more 
+
+echo
+echo This script does all the installation described in the README in an automated way.
+echo But be sure to have read the \"Before anything else\" section.
+echo
+
+read -s -p "To continue, press enter. Otherwise, press CTRL+C" 
+echo
+
+echo Downloading and installing requirements...
+
+# Installing requirements
+sudo apt-get update
+sudo apt-get install build-essential gcc flex bison libexpat-dev libpq-dev
+sudo apt-get install perl=5.18.2-2ubuntu1
```

Comment:
To be explicit, this seems like the sort of thing that is done in order to exploit a known bug. If that is not the case, then what reason do you have for doing this? Why not just install 'perl'?